dora/integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -156,7 +157,15 @@ private void umountInternal() {
       }
     } else {
       try {
-        exitCode = new ProcessBuilder("fusermount", "-u", "-z", mountPath).start().waitFor();
+        Process process = new ProcessBuilder("fusermount", "-u", "-z", mountPath).start();
+        boolean exited = process.waitFor(UNMOUNT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
```

Comment:
`Ten seconds` here is a long time.
If this happens, admin/user might loses patients and press CTRL+C.
So, it's better to print some warning message before the waiting.

Reply:
Do you think the same timeout wait is needed for the `umount` on Line 177?